packages/modular-scripts/src/esbuild-scripts/api.ts

```diff
@@ -50,10 +61,16 @@ ReactDOM.render(<Component />, DOMRoot);`;
           });
         },
       },
-      createRewriteDependenciesPlugin({
-        ...dependencies,
-        'react-dom': dependencies.react,
-      }),
+      createRewriteDependenciesPlugin(
+        {
+          ...dependencies,
+          'react-dom': dependencies.react,
```

Comment:
Surely `react-dom` must be a dependency in the tree for starting a view to work, so we should use the resolution version from the repo in the case where `react-dom` is a different version to `react`?

Reply:
Done.